lib/apr/views/seller_slack_view.ex

```diff
@@ -7,7 +7,10 @@ defmodule Apr.Views.InvoiceSlackView do
 
     cond do
       routing_key =~ "merchantaccount" ->
-        merchant_account_message(event, partner_data)
+        case event["verb"] do
```

Comment:
The routing key will be `merchantaccount.external_account_restricted_soon`, so we can probably move this condition higher up in the `routing_key` check on line 9.

Reply:
True! Addressed it here: https://github.com/artsy/aprd/pull/364/commits/b3df5adef2eba89f6c233503a59636f3f4d3534f